diff --git a/build/src/utils/npm.js b/build/src/utils/npm.js
--- a/build/src/utils/npm.js
+++ b/build/src/utils/npm.js
@@ -171,7 +171,7 @@
     transformStream.on("data", (chunk)=>{
         hash.update(chunk);
     });
-    await pipeline(stream, transformStream, _tar().default.extract({
+    await pipeline(stream, transformStream, (_tar().default || _tar()).extract({
         cwd,
         filter,
         onentry: (0, _createFileTransform.createEntryResolver)(name),
diff --git a/build/src/utils/tar.js b/build/src/utils/tar.js
--- a/build/src/utils/tar.js
+++ b/build/src/utils/tar.js
@@ -86,7 +86,7 @@
     debug(`Extracting ${input} to ${output} using JS tar module`);
     // tar node module has previously had problems with big files, and seems to
     // be slower, so only use it as a backup.
-    await _tar().default.extract({
+    await (_tar().default || _tar()).extract({
         file: input,
         cwd: output
     });
